src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -3,70 +3,35 @@ import Logger from "bunyan";
 import { NextFunction, Request, Response } from "express";
 import { getJWTRequest, TokenType, validateQsh } from "~/src/jira/util/jwt";
 import { Installation } from "~/src/models/installation";
-import { moduleUrls } from "~/src/routes/jira/atlassian-connect/jira-atlassian-connect-get";
+import { genericContainerActionUrls, moduleUrls } from "~/src/routes/jira/atlassian-connect/jira-atlassian-connect-get";
 import { matchRouteWithPattern } from "~/src/util/match-route-with-pattern";
 import { fetchAndSaveUserJiraAdminStatus } from "middleware/jira-admin-permission-middleware";
-import { booleanFlag, BooleanFlags } from "config/feature-flags";
+import { envVars } from "~/src/config/env";
 
 export const jiraSymmetricJwtMiddleware = async (req: Request, res: Response, next: NextFunction) => {
-	const authHeader = req.headers["Authorization"] as string;
+	const authHeader = req.headers["authorization"] as string;
```

Comment:
Should we take both?

Reply:
Harminder can confirm this but I think he said that the header in provider bridge is all lowercase. Wouldn't hurt to accept both though.

Reply:
This change isn't needed/doesn't matter as it's case insensitive. This is a HTTP convention that Node.js follows. We can access header values using different case variations, and it will still retrieve the same value.

Reply:
As a sanity check I just modified the property in one of the tests, replacing 'authorization' with 'Authorization' and then 'AUTHORIZATION' (all 3 are valid casings for the property).

<img width="1075" alt="Screenshot 2023-06-20 at 3 27 23 pm" src="https://github.com/atlassian/github-for-jira/assets/37155488/4ecb4555-8960-4e68-9e40-45b1a56e214b">

<img width="1068" alt="1" src="https://github.com/atlassian/github-for-jira/assets/37155488/03e5454a-5b3e-4563-a932-9902210dfce0">